CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ### Prs in Release
```

Comment:
Can you remove everything above your entry (`## 0.7.0`) in the `CHANGELOG.md`? After that I'll merge. Thanks.

Reply:
Just need to remove the below lines so your entry is the first one.

```diff
- ## 0.7 (Not Releases)
-
- ### Prs in Release

## 0.7.0
```